### ISSUE TITLE
Support Squirrel flag JSON output for SUFeedURL - Sparkle

### ISSUE THREAD

**Issue by rishimodha** [microsoft/appcenter#1304]
**Describe the solution you'd like**
Support the Squirrel flag for the Sparkle update feed. This was supported on HockeyApp and it is required for Electron AutoUpdater, which does not support XML output and requires JSON. See example: https://rink.hockeyapp.net/api/2/apps/bc02347d258a40e2af46cc8de3566bab.json?squirrel=true

**Describe alternatives you've considered**
It won't be possible for us to complete the transition to AppCenter from HockeyApp until support for Squirrel JSON format is added. Please do not force a transition before this is supported as it will cut off our customer's ability to update their apps.

**Comment by Ajaykn21** [comment]
Hi @rishimodha,

Thanks for bringing up this issue to our notice.
This was of low priority when we started sparkle feed support in AppCenter.

Now, we have added the support of Squirrel JSON format.
Here is the corresponding AppCenter sparkle feed url for your example.

> https://api.appcenter.ms/v0.1/public/sparkle/apps/<app_secret>?squirrel=true
This returns the latest release in JSON format and if there is no release then returns empty object.

>  https://api.appcenter.ms/v0.1/public/sparkle/apps/<app_secret>
This as usual returns the latest 5 releases in xml format.

Please let us know your experience on this and get back to us if you have any other concerns/requirements.

**Comment by rishimodha** [comment]
@Ajaykn21 thanks for adding this. 

If possible, can you add handling to redirect the HockeyApp URL that contains the squirrel tag to the corresponding AppCenter URL with the squirrel tag?

**Comment by Ajaykn21** [comment]
Yeah @rishimodha, we are working on this already and will update over here once it is ready.

**Comment by rishimodha** [comment]
@Ajaykn21 - Any update on this?

**Comment by Ajaykn21** [comment]
@rishimodha , we are flooded with other priority requests. We have started working on this and will update you once done.

**Comment by Ajaykn21** [comment]
@rishimodha, we have released this to Production.
And we redirect HockeyApp Sparkle feed to AppCenter Sparkle feed for the apps that are cutovered.
Although, it is recommended to update your apps SUFeedUrl to AppCenter Sparkle feed url.

Can you please check and let us know.

**Comment by rishimodha** [comment]
@Ajaykn21 Yes, this working great. Thank you!

**Comment by ajaykn** [comment]
@rishimodha can we close this issue, as this is already fixed ?
